src/lib/utils.ts

```diff
@@ -18,6 +18,17 @@ import {
 } from '../codec/ibc/lightclients/tendermint/v1/tendermint';
 import { PublicKey as ProtoPubKey } from '../codec/tendermint/crypto/keys';
 
+// Subset of methods from winston.js
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export interface Logger {
+  error: (message: string, ...meta: any[]) => Logger;
```

Comment:
I copied this from WinstonJS, but it would be good to document how these expected to be logged. (just copy the relevent lines from winstonjs)

Reply:
I'm not sure what you mean here. Are you talking about [logging levels](https://github.com/winstonjs/winston/tree/v3.3.3#logging-levels)?

Reply:
Sorry, I meant `...meta: any[]`.

Reply:
I think this is an example of using those: https://github.com/winstonjs/winston#string-interpolation